pkg/connector/tickets.go

```diff
@@ -325,12 +325,27 @@ func (j *Jira) ListTicketSchemas(ctx context.Context, p *pagination.Token) ([]*v
 		return nil, "", nil, wrapError(err, "failed to get projects")
 	}
 
+	filteredProjects := projects
```

Comment:
I am looking into using a filter on the projects find actually

Reply:
I took a look at this, and while the underlying API call does support a keys field, the SDK does not support, but it is a forked version. Maybe since this blocking tide we can get this in. I will look at this tomorrow and see about adding the support for keys in.

Reply:
There's also a projects GetAll but it returns something entirely differently.